nic_operator_helm/nic_operator_helm_ci_test.sh

```diff
@@ -13,11 +13,21 @@ export POLL_INTERVAL=${POLL_INTERVAL:-10}
 
 export KUBECONFIG=${KUBECONFIG:-/etc/kubernetes/admin.conf}
 
+export SRIOV_NETWORK_OPERATOR_REPO=${SRIOV_NETWORK_OPERATOR_REPO:-https://github.com/k8snetworkplumbingwg/sriov-network-operator.git}
+export SRIOV_NETWORK_OPERATOR_BRANCH=${SRIOV_NETWORK_OPERATOR_BRANCH:-''}
+export SRIOV_NETWORK_OPERATOR_PR=${SRIOV_NETWORK_OPERATOR_PR:-'97'}
```

Comment:
Do we need really default value here?

Reply:
Not anymore, i just found out that the PR was merged, and test-e2e-k8s rule is not in master

Reply:
Removed it in latest update